website/content/docs/release-notes/consul-terraform-sync/v0_6_0_beta.mdx

```diff
@@ -15,6 +15,7 @@ description: >-
 - New CLI Autocompletion
 - Addition of Features to Support HCP Consul <EnterpriseAlert inline />
 - Addition of Features to Support Terraform Cloud Agents <EnterpriseAlert inline />
+- Get all tasks API endpoint
```

Comment:
nit: The capitalization doesn't match the rest of the items in this list.

Reply:
```suggestion
- Get All Tasks API Endpoint
```